.github/workflows/actions.yml

```diff
@@ -0,0 +1,24 @@
+name: Dolos CI/CD
+
+on: [pull_request, issues, push]
```

Comment:
do we need to add the entry named `issue` there ?

also, please sync your pr with the master branch.

Reply:
removed issue entry point. Please check!
Also my PR synced with master branch